index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="header-logo">Mihreteab</div>
+        <img class="only-mobile" src="./Images/Union.svg" alt="Union vectors">
+        <nav class="only-desktop">
+            <ul>
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+            </ul>
+        </nav>
+    </header>
+
+    <section id="headline-section">
+        <h1>I’m Mihreteab Glad to see you!</h1>
+
+        <article>
+            <p>
+                I’m a software developer! 
+                I can help you build a product , feature or website Look 
+                through some of my work and experience! If you like what 
+                you see and have a project you need coded, don’t hestiate 
+                to contact me.
+            </p>
+        </article>
+
+        <h4>LET'S CONNECT</h4>
```

Comment:
- [ ] I think it would be a good idea to use the `<h2>` tag instead of the `<h4>` because it's considered good practice to follow an order when using `<hX>` tags. But you could even use a `<p>` and add a custom style because this text it's not like a title or header to use the `<hX>` tag. Please use and `<h2>` or `<p>` tag for this text :smiley: